### ISSUE TITLE
no Objects

### ISSUE THREAD

**Issue by mcbirne** [unltdnetworx/ioBroker.energymanager#37]
Hello,
I install the energymanager adapter. I add the IP address and the password. The adapter is green. But I only have the Object manageReboot.
I use the energymanager flex from Solarwatt.

**Comment by mcbirne** [comment]
Information from the logfile 
![image](https://user-images.githubusercontent.com/38691204/154490241-504a0fad-5782-4075-b9bb-d8125f5dd006.png)

**Comment by unltdnetworx** [comment]
This datapoint is also created without any connection, therefore it appears.

Do you reach the web interface of the energymanager inside of a browser with this ip?

**Comment by mcbirne** [comment]
If I open http://192.168.178.124 in my browser I come to the gui from the energymanager. I also can ping the IP Adress from the IObroker ping Adapter.

**Comment by unltdnetworx** [comment]
http://192.168.178.124/rest/kiwigrid/wizard/devices
Kommt hier JSON-Code an? Bitte nicht posten.
In den Adaptereinstellungen steht aber nur die IP Adresse ohne "http://", richtig?

**Comment by mcbirne** [comment]
Nein, es kommt {"error":{"message":"HTTP 404 Not Found","http-code":404,"exception":{"class":"javax.ws.rs.NotFoundException","message":"HTTP 404 Not Found","localized-message":"HTTP 404 Not Found"}}}
Ich habe es im Adapter ohne http eingetragen. Nur die IP Adresse.
Sollte der Adapter mit dem energimanager flex von Solarwatt funktionieren?

**Comment by unltdnetworx** [comment]
Das sieht nicht gut aus. Ich kenne das Gerät leider nicht und kann daher nichts dazu sagen.
Dazu bräuchte ich den HTML-Code des Webinterface. Vllt. kann man da etwas rauslesen.

**Comment by mcbirne** [comment]
Wo bekomme ich den Code her? 
Ich habe den Hersteller Solarwatt mal angeschrieben und nach der API gefragt.

**Comment by unltdnetworx** [comment]
Es gibt leider keine API, daher der Adapter.

Der Quellcode der Seite wäre mit einem Browser auslesbar. Dazu einfach einloggen und Rechtsklick "Quellcode anzeigen" klicken.

Den Code bitte nicht hier posten, es könnten persönliche Info enthalten sein.

**Comment by unltdnetworx** [comment]
Am Besten an die e-Mailadresse, welche in der Adapterbeschreibung steht.

**Comment by mcbirne** [comment]
Schade. Ich habe dir mal die Mail geschickt. Vielleicht gibt es ja eine Möglichkeit
Ich habe aktuell die Werte noch gar nicht gefunden. Im Webportal ist alles vorhanden, aber in der Box finde ich sie nicht. 
Ich habe mal den Anbieter angeschrieben um zu erfahren wo die Werte in der Box sind.

**Comment by unltdnetworx** [comment]
Ich habe den Quellcode gerade gesehen. Der wird leider fast vollständig mit JavaScript on-the-fly generiert und ist daher leider nicht sichtbar.
Ohne das Gerät, kann ich da leider auch nichts testen.

Leider ist der Adapter somit nicht kompatibel mit dem Manager flex.

**Comment by mcbirne** [comment]
Schade. Danke für die Mühe.
